db/migrate/20220113234632_change_email_of_users_uniqe.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeEmailOfUsersUniqe < ActiveRecord::Migration[6.1]
+  def change
+    add_index :users, :email, unique: true
```

Comment:
This looks good, although you should not add a migration. You can edit the `create_users` migration directly as it corresponds to the same commit.